lib/commands/general.js

```diff
@@ -22,7 +22,8 @@ commands.mobilePerformEditorAction = async function (opts = {}) {
 
 commands.mobileSwipe = async function (opts = {}) {
   const {direction, element} = assertRequiredOptions(opts, ['direction', 'element']);
-  return await this.espresso.jwproxy.command(`/appium/execute_mobile/${element}/swipe`, 'POST', {direction});
+  let elementId = element.ELEMENT || element;
```

Comment:
please use `util.unwrapElement` from appium-support

Reply:
also, this could be spelled like

`return await this.espresso.jwproxy.command(`/appium/execute_mobile/${util.unwrapElement(element)}/swipe`, 'POST', {direction})`

Reply:
I put in a PR to make unwrapElement also unwrap W3C elements: https://github.com/appium/appium-support/pull/93/files

Reply:
I'd also go through the codebase to make sure this helper is used everywhere instead of `manual` unwrapping

Reply:
@mykla-mokhnach should i wait for the unwrapElement which @dpgraham implemented to get merged ?

Reply:
No need to wait. It will get the updated driver later.